Intersect.Server/Web/RestApi/Types/Logs/IpAddress.cs

```diff
@@ -0,0 +1,10 @@
+namespace Intersect.Server.Web.RestApi.Types.Logs;
+
+public partial class IpAddress
```

Comment:
This needs to be ResponseBody, RequestBody (or Payload if used for both), it's horrible that we ever named this IpAddress

Why is it horrible? Because this exists: https://learn.microsoft.com/en-us/dotnet/api/system.net.ipaddress?view=net-9.0